packages/react-native/scripts/generate.js

```diff
@@ -135,7 +135,7 @@ function generate({ configPath, absolute = false, useJs = false }) {
     updateView(global.view, annotations, normalizedStories, ${options});
   }
 
-  export const view = global.view;
+  export const view: ReturnType<typeof start> = global.view;
```

Comment:
To support the useJs option what about adding something like this?

```suggestion
  export const view${useJs ? '' : ': ReturnType<typeof start>'} = global.view;
```

Reply:
oops, missed useJs check. Fixed now